src/Refitter.Core/RefitGeneratorSettings.cs

```diff
@@ -140,6 +140,13 @@ public class RefitGeneratorSettings
     [JsonPropertyName("optionalParameters")]
     [JsonProperty("optionalParameters")]
     public bool OptionalParameters { get; set; }
+    
+    /// <summary>
+    /// Gets or sets the relative path to which the output files are generated. (default: ./Generated)
+    /// </summary>
+    [JsonPropertyName("output")]
```

Comment:
@Noblix After giving it a bit more thought, maybe we should rename the JSON propert name from `output` to `outputFolder` so it explicitly states in the name that this is not the output filename but the output folder